src/app/components/dashboard-company-selectbox/dashboard-company-selectbox.coffee

```diff
@@ -23,7 +23,8 @@ DashboardCompanySelectboxCtrl = ($scope, $state, $stateParams, $uibModal, MnoeCu
     # Switch to selected company
     MnoeAppInstances.emptyAppInstances()
     MnoeAppInstances.clearCache()
-    MnoeOrganizations.get(organization.id)
+    MnoeOrganizations.get(organization.id).then( (org) ->
+      if !org.organization.active then $state.go "home.company")
```

Comment:
There is an empty space before `(`, `if not` can be `unless`, and the the indentation should be:
```
MnoeOrganizations.get(organization.id).then((org) ->
  $state.go('home.company') unless org.organization.active
)
```